integritybackend/file_util.py

```diff
@@ -221,7 +221,7 @@ def decrypt(self, key, file_path, dec_file_path):
                 dec.write(cipher.decrypt(prev_data))
 
     @staticmethod
-    def digest_cidv1(self, file_path):
+    def digest_cidv1(file_path):
```

Comment:
@staticmethod does not set self param creating error.

Reply:
@makeworld-the-better-one how was this tested?